src/node.cc

```diff
@@ -981,6 +983,42 @@ void SetupNextTick(const FunctionCallbackInfo<Value>& args) {
       FIXED_ONE_BYTE_STRING(args.GetIsolate(), "_setupNextTick"));
 }
 
+void PromiseRejectCallback(PromiseRejectMessage message) {
+  Isolate* isolate = Isolate::GetCurrent();
```

Comment:
Is it really going to be deprecated? Sometimes there is no other way. In this case I think we can use `promise->GetIsolate()`